unetsocket/c/unet.h

```diff
@@ -5,6 +5,18 @@
 
 typedef void *unetsocket_t;        ///< unet socket connection
 
+/// Maximum length of a frame ID string
+
+#define FRAME_ID_LEN        64
+
+/// Transmit sampling rate
+
+#define TXSAMPLINGFREQ           192000
```

Comment:
Should this be a part of `unet.h`? This number isn't correct if using `Unet audio`. Might be safer to read it from `phy.dacrate`.

Reply:
Removed `TXSAMPLINGFREQ` now. In the sample programs, I am now using `phy.dacrate` to use the correct sampling rate.